src/main/java/net/mcreator/ui/dialogs/AcceleratorDialog.java

```diff
@@ -0,0 +1,171 @@
+				BorderFactory.createLineBorder((Color) UIManager.get("MCreatorLAF.BRIGHT_COLOR"), 1),
+				L10N.t("dialog.accelerators.accelerators"), 0, 0, getFont().deriveFont(12.0f),
+				(Color) UIManager.get("MCreatorLAF.BRIGHT_COLOR")));
+		sections.addActionListener(e -> cl.show(sectionsPanel, (String) sections.getSelectedItem()));
+
+		AcceleratorsManager.INSTANCE.SECTIONS.forEach(sectionName -> {
+			List<JComponent> comps = new ArrayList<>();
+			actions.stream().filter(action -> action.getAccelerator().getSection().equals(sectionName))
+					.forEach(action -> {
+						KeyStroke keyStroke = action.getAccelerator().getKeyStroke();
+						if (keyStroke != null) {
+							comps.add(new JLabel(action.getName()));
+							JAcceleratorButton button = new JAcceleratorButton(setButtonText(keyStroke),
+									action.getAccelerator());
+							comps.add(button);
+							buttonsList.add(button);
+						}
+					});
+
+			sectionsPanel.add(new JScrollPane(PanelUtils.pullElementUp(
+							PanelUtils.gridElements(comps.size() / 2, 2, 5, 5, comps.toArray(new JComponent[0])))),
+					L10N.t("dialog.accelerators.section." + sectionName));
+		});
+
+		JPanel buttonsPanel = new JPanel(new FlowLayout(FlowLayout.CENTER));
+
+		JButton confirm = new JButton(L10N.t("common.confirmation"));
+		confirm.addActionListener(a -> {
+			AcceleratorsManager.INSTANCE.loadAccelerators(mcreator.actionRegistry);
+			AcceleratorDialog.this.setVisible(false);
+		});
+		buttonsPanel.add(confirm);
+
+		JButton resetAll = new JButton(L10N.t("dialog.accelerators.reset_all"));
+		resetAll.addActionListener(a -> {
+			int n = JOptionPane.showConfirmDialog(mcreator.actionRegistry.getMCreator(),
+					L10N.t("dialog.accelerators.reset_all.message"), L10N.t("common.confirmation"),
+					JOptionPane.YES_NO_OPTION, JOptionPane.QUESTION_MESSAGE);
+			if (n == JOptionPane.YES_OPTION) {
+				AcceleratorsManager.INSTANCE.setDefaultValues(mcreator.actionRegistry);
+				// We open a new dialog, so we don't need to iterate every button to change their value
+				setVisible(false);
+				new AcceleratorDialog(mcreator);
+			}
+		});
+		buttonsPanel.add(resetAll);
+
+		JButton cancel = new JButton(L10N.t("common.close"));
+		cancel.addActionListener(a -> {
+			setVisible(false);
+			buttonsList.forEach(button -> AcceleratorsManager.INSTANCE.setInCache(button.getAcceleratorID(),
+					button.getKeyStroke()));
+		});
+		buttonsPanel.add(cancel);
+
+		add(PanelUtils.northAndCenterElement(PanelUtils.northAndCenterElement(new JEmptyBox(), northPanel, 5, 5),
+				PanelUtils.centerAndSouthElement(sectionsPanel, buttonsPanel, 20, 20), 20, 20));
+
+		setSize(700, 750);
+		setResizable(true);
+		setLocationRelativeTo(mcreator);
+		setVisible(true);
+	}
+
+	public static String setButtonText(KeyStroke keyStroke) {
+		String acceleratorText = "";
+		int modifiers = keyStroke.getModifiers();
+		if (modifiers > 0) {
+			acceleratorText = InputEvent.getModifiersExText(modifiers);
+			acceleratorText += " + ";

```

Comment:
No need to split this assignment into two lines.

Reply:
Klemen already split it before. This code is 100% copy and paste of the old code.

Reply:
Then this would be an improvement of Klemen's code.

Reply:
Not sure it really improves the code, but ok.